src/utils/canvas.ts

```diff
@@ -24,3 +24,54 @@ export async function drawBlobOnCanvas(
 export async function canvasToBlob(canvas: HTMLCanvasElement, mimeFormat: MimeType): Promise<Blob> {
   return await new Promise(resolve => canvas.toBlob(resolve, mimeFormat));
 }
+
+export async function getImageProperties(originalBlob: Blob): Promise<ImageProperties> {
+  let imgObjectUrl: any;
+  const imgCanvas = document.createElement('canvas');
+  try {
+    const imgCtx = imgCanvas.getContext('2d');
+    imgObjectUrl = window.URL.createObjectURL(originalBlob);
+    const img: any = await new Promise((resolve, reject) => {
+      const img = new Image();
+      img.onload = () => resolve(img);
+      img.onerror = reject;
+      img.src = imgObjectUrl;
+    });
+    imgCanvas.width = img.width;
+    imgCanvas.height = img.height;
+    imgCtx.drawImage(img, 0, 0);
```

Comment:
tried to use `drawBlobOnCanvas()` but I didn't want to change it to return img width and height (because then I would have to check if that's ok with all the other functions that use `drawBlobOnCanvas()` and it's just a bit too much out of scope for this MR I think) so I just left the duplicated code for putting the blob on canvas here.